frontend/document/credocument.lua

```diff
@@ -714,12 +714,12 @@ function CreDocument:getTextFromPositions(pos0, pos1, do_not_draw_selection)
         drawSelection, drawSegmentedSelection)
     logger.dbg("CreDocument: get text range", text_range)
     if text_range then
-        -- local line_boxes = self:getScreenBoxesFromPositions(text_range.pos0, text_range.pos1)
+        local line_boxes = self:getScreenBoxesFromPositions(text_range.pos0, text_range.pos1)
```

Comment:
getScreenBoxesFromPositions() may be expensive, wondering if there are other calls to getTextFromPositions() that would not need it, and then if calling it and returning sboxes may be a parameter to that getTextFromPositions().

Reply:
Yes, there are only 2 callers that do not require it:
https://github.com/koreader/koreader/blob/5d68ece781519245850b8d011a56de109f70738b/frontend/apps/reader/modules/readerhighlight.lua#L1847
https://github.com/koreader/koreader/blob/5d68ece781519245850b8d011a56de109f70738b/frontend/apps/reader/modules/readerview.lua#L1407
I thought the parameter was not needed.